Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -0,0 +1,2539 @@
+//
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import QuartzCore
+import ArcGIS
+
+// MARK: - Time Slider Control
+
+/*
```

Comment:
@njarecha You could make these comments viewable in Xcode's quick help by adding a second `*`, or by using `///`. The easiest way to make these documentation comments is with the Add Documentation (⌥⌘/) keyboard shortcut. That shortcut is extremely helpful for documenting functions as it adds placeholders for parameters and the return value.

Reply:
fixed